src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+intersection_of: MONDO:0700097 MONDO:0019169 {source="OMIA:001406"} ! cross-species analog pyruvate dehydrogenase deficiency
+
+[Term]
+id: MONDO:1010106
+name: multiple acyl-coa dehydrogenase deficiency, non-human animal
+def: "Multiple acyl-coa dehydrogenase deficiency that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001457 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001457"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009282 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog multiple acyl-CoA dehydrogenase deficiency
+
+[Term]
+id: MONDO:1010107
+name: acute intermittent porphyria, non-human animal
+def: "Acute intermittent porphyria that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001493 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001493"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001493"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008294 {source="OMIA:001493"} ! cross-species analog acute intermittent porphyria
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+
+[Term]
+id: MONDO:1010108
+name: glycogen storage disease IIIa, non-human animal
+def: "Glycogen storage disease IIa that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001577 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001577"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001577"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009291 {source="OMIA:001577"} ! cross-species analog glycogen storage disease III
+
+[Term]
+id: MONDO:1010109
+name: dihydropyrimidinase deficiency, non-human animal
+def: "Dihydropyrimidinase deficiency that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001776 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001776"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001776"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009111 {source="OMIA:001776"} ! cross-species analog dihydropyrimidinuria
+
+[Term]
+id: MONDO:1010110
+name: hyperphosphatemia, non-human animal
+def: "Hyperphosphatemia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002052 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002052"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0000328 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog hyperphosphatemia
+
+[Term]
+id: MONDO:1010111
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.